docs/analytics/aggregate.md

```diff
@@ -1,15 +1,20 @@
-# Using the Analytics APIs to access call performance aggregate data
+# Call Aggerate Reports
 
-## Composing a request to the Call Performance Aggregate API
+It is quite common to acquire and segment analytics data. Call Performance Aggregate API provides ability to aggregate call realted data in various ways such as by summing, averaging, getting minimum or maximum or percetage of data etc. For more information regarding common use cases, please refer to the [overview section](index.md) and for detailed information regarding how to compose API Request, please refer to the data guide below.
```

Comment:
Please review this content for grammar and spelling. Install Grammarly or some other checker to ensure that sentence structure makes sense, and all words are spelled correctly.

Reply:
ok